submissions/Yasya23/holi-style-popup/index.html

```diff
@@ -0,0 +1,209 @@
+            <img
+              src="icons/popup-button.svg"
+              alt="Popup icon"
+              width="30"
+              height="30"
+              class="navigation__popup-icon"
+            />
+          </label>
+          <input type="checkbox" id="popup" class="popup__input" />
+          <div class="popup__container">
+            <ul class="popup__list">
+              <li class="popup__box-img">
+                <a href="#">
+                  <img
+                    src="icons/Collection.svg"
+                    alt="Collection icon"
+                    width="50"
+                    height="50"
+                    class="popup__img"
+                  />
+                </a>
+                <div>Collection</div>
+              </li>
+              <li class="popup__box-img">
+                <div>
+                  <a href="#">
+                    <img
+                      src="icons/mail.svg"
+                      alt="Mail icon"
+                      width="50"
+                      height="50"
+                      class="popup__img"
+                    />
+                  </a>
+                </div>
+                <div>Mail</div>
+              </li>
+              <li class="popup__box-img">
+                <a href="#">
+                  <img
+                    src="icons/Contacts.svg"
+                    alt="Contacts icon"
+                    width="50"
+                    height="50"
+                    class="popup__img"
+                  />
+                </a>
+                <div>Contacts</div>
+              </li>
+              <li class="popup__box-img">
+                <a href="#">
+                  <img
+                    src="icons/facebook.svg"
+                    alt="Facebook icon"
+                    width="50"
+                    height="50"
+                    class="popup__img"
+                  />
+                </a>
+                <div>Facebook</div>
+              </li>
+              <li class="popup__box-img">
+                <a href="#">
+                  <img
+                    src="icons/files.svg"
+                    alt="File icon"
+                    width="50"
+                    height="50"
+                    class="popup__img"
+                  />
+                </a>
+                <div>Files</div>
+              </li>
+              <li class="popup__box-img">
+                <a href="#">
+                  <img
+                    src="icons/Friends.svg"
+                    alt="Friends icon"
+                    width="50"
+                    height="50"
+                    class="popup__img"
+                  />
+                </a>
+                <div>Friends</div>
+              </li>
+              <li class="popup__box-img">
+                <a href="#">
+                  <img
+                    src="icons/instagram.svg"
+                    alt="Instagram icon"
+                    width="50"
```

Comment:
Get back to it. What is the reason to use inline styles here?

Reply:
I realized that in this case, it does not make sense. This is useful when reserving space for images is necessary so there are no jumps during page loading. I removed these characteristics and moved them to CSS.